earhorn/archive.py

```diff
@@ -2,13 +2,64 @@
 from subprocess import DEVNULL, run
 from threading import Thread
 from typing import Optional
+import boto3
+import csv
+from dotenv import load_dotenv
+from botocore.exceptions import ClientError
+import os
+
+load_dotenv()
 
 from loguru import logger
 
 from ._ffmpeg import FFMPEG
 
 TIMESTAMP_FORMAT = "%Y%m%d_%H%M%S"
 
+def upload_file(file_name, bucket, object_name=None):
+    """Upload a file to an S3 bucket
+
+    :param file_name: File to upload
+    :param bucket: Bucket to upload to
+    :param object_name: S3 object name. If not specified then file_name is used
+    :return: True if file was uploaded, else False
+    """
+
+    # If S3 object_name was not specified, use file_name
+    if object_name is None:
+        object_name = file_name
+
+    # Upload the file
+    s3_client = boto3.client('s3')
+    try:
+        response = s3_client.upload_file(file_name, bucket, Key="{}".format(os.path.basename(file_name)))
+    except ClientError as e:
+        return False
+    return True
```

Comment:
Please always log errors, at least as debug or warning message  if you think the error is not important.